src/graphql/data/nft/Asset.ts

```diff
@@ -117,22 +116,46 @@ export function useAssetsQuery(
   last?: number,
   before?: string
 ) {
-  const queryData = useLazyLoadQuery<AssetQuery>(assetQuery, {
-    address,
-    orderBy,
-    asc,
-    filter,
-    first,
-    after,
-    last,
-    before,
+  const queryResult = useQuery(assetQuery, {
+    variables: {
+      address,
+      orderBy,
+      asc,
+      filter,
+      first,
+      after,
+      last,
+      before,
+    },
   })
-  const { data, hasNext, loadNext, isLoadingNext } = usePaginationFragment<AssetPaginationQuery, any>(
-    assetPaginationQuery,
-    queryData
-  )
+  const { loading, data, fetchMore, startPolling } = queryResult
+
+  startPolling(999)
```

Comment:
see above, i recommend `pollInterval`, but regardless, this should probably be called in a useEffect

```
useEffect(() => { startPolling(999) }, [])
```

Reply:
also nit. can use the ms macro to format intervals

```
ms`10s`
```